docs/ExposePortsToInternet.md

```diff
@@ -0,0 +1,46 @@
+<!--- master only -->
+> ![warning](images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Expose Container Port to Host-Interface / Internet
+
+In the [Calico without Docker networking tutorial](calico-with-docker/without-docker-networking/README.md) 
+or the [Calico as a Docker network plugin tutorial](calico-with-docker/docker-network-plugin/README.md)
+we created containers and assigned endpoints (a container interface) to them. This is used for Container-
+To-Container communication.
+
+The example below shows how to expose a port of a container to the Host-Interface so this container is 
+reachable from outside / the internet.
+
+## Why isn't the `-p` flag on `docker run` working as expected?
+If you connect containers to the docker0 interface, then calico would not be able to enforce security rules
+between workloads; all containers on the bridge would be able to communicate with each other.
+
+## How can i make a container reachable from the Internet (Host-Interface)?
```

Comment:
Instead of having this as a question, it'd be nice to have it as a goal, such as "Making a Container Reachable by the Internet".